app/Policies/UserPolicy.php

```diff
@@ -28,9 +28,9 @@ public function viewAny(User $user): bool
     {
         return
             $user->hasAnyRoleBase([
-                Role::STAFF, Role::SECRETARY, Role::DIRECTOR, Role::WORKSHOP_ADMINISTRATOR, Role::WORKSHOP_LEADER
+                Role::STAFF, Role::SECRETARY, Role::DIRECTOR, Role::WORKSHOP_ADMINISTRATOR, Role::WORKSHOP_LEADER, Role::STUDENT_COUNCIL_SECRETARY
             ])
-            || $user->isPresident();
+            || $user->isPresident() || $user->hasRole(Role::STUDENT_COUNCIL, Role::SCIENCE_VICE_PRESIDENT);
```

Comment:
You might want to add the other vice president also. Create an isStudentCouncilLeaders (or similar) function for this.